src/ir/types/BUILD

```diff
@@ -37,6 +37,7 @@ cc_test(
         "//src/common/logging",
         "//src/common/testing:gtest",
         "//src/ir",
+        "//src/ir/proto:types",
```

Comment:
Should we also remove the `//third_party/arcs/proto:manifest_cc_proto` dependency from the `:types` library?

(Sorry for the odd placement, GitHub won't let me put comments outside of the context it draws around modified lines).